Makefile

```diff
@@ -85,13 +88,20 @@ test: ## run test
 FORCE:
 
 # Build a binary from a cmd.
-bin/%: cmd/% FORCE
+$(COMMAND_BIN): bin/%: cmd/% FORCE
```

Comment:
This is a "pattern rule" and I used [this](http://ynonperek.com/course/make/patterns.html) as a guide